tests/testthat/test-tfrmt_checks.R

```diff
@@ -40,3 +40,77 @@ test_that("Testing error messages for missing argument col_style_structure",{
     ),
     "Column element is missing from col_style_structure[.] Note: col here refers to the values within the column variable in your data, rather than the variable name itself")
 })
+
+
+
+test_that("Testing error message for invalid input to plan parameters, specifically for col_style_plan",{
```

Comment:
for both of these new tests, can you please add the expected error as well? Similar to the test above

Reply:
Due to the revision made as a result of https://github.com/GSK-Biostatistics/tfrmt/pull/428#discussion_r1480020661 , expected error is:
![image](https://github.com/GSK-Biostatistics/tfrmt/assets/72526731/393cdb2c-2d31-4978-add3-85ea19d88ec7)
When this message is defined as expected in test (line 73):
![image](https://github.com/GSK-Biostatistics/tfrmt/assets/72526731/35c17a0f-7967-4ae6-a4cd-cce7cf684aab)
The test fails:
![image](https://github.com/GSK-Biostatistics/tfrmt/assets/72526731/f629e2f4-67e1-4bb8-8a2e-44a988e6a433)

Hence, a workaround was to use tryCatch() instead of expect_error() - as done in https://github.com/GSK-Biostatistics/tfrmt/pull/428/commits/f7c0b5a49c4aed3dd0b5c22d13d4b24c607d3cc2